pkg/apis/extensions/types.go

```diff
@@ -35,6 +35,15 @@ import (
 	"k8s.io/kubernetes/pkg/util/intstr"
 )
 
+// The following identify resource constants for Kubernetes object types
+const (
+	// ReplicaSets, number
+	ResourceReplicaSets api.ResourceName = "replicasets"
```

Comment:
@derekwaynecarr I believe we can relax the present of `/` as if the name is not in the standard resource list it will be invalid anyway.